src/app/admin/manage-deposits/manage-deposits.component.html

```diff
@@ -33,14 +33,16 @@
         <div *ngIf="(unapprovedDeposits$ | async) as unapprovedDeposits else spinner" class="col-md-12">
             <ul class="list-group mt-3">
                 <li *ngFor="let deposit of unapprovedDeposits; let i = index"
-                    class="list-group-item gray-text">
+                    class="list-group-item gray-text link-remove-style">
                     <span class="float-left explainer-mini">
                         {{ deposit.deposit.reference }} ({{ deposit.user.email }})
                     </span>
-                    <span (click)="declineDeposit(deposit.deposit.id).subscribe()" class="float-right cursor-pointer">
+                    <a [routerLink]="">
+                    <span (click)="declineDeposit(deposit.deposit.id).subscribe()" class="float-right">
                         <i class="fas fa-trash-alt"></i>
                     </span>
-                    <a class="float-right mr-4 cursor-pointer" href="mailto:{{ deposit.user.email }}">
+                    </a>
```

Comment:
Click event should be always added to an anchor tag. This leads to breaking the semantics. Is there a visual reason for that?

Reply:
OK thnx, will fix it.